cromwell_tools/scripts/cromwell-tools

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python2.7
+
+import argparse
+import json
+from cromwell_tools import cromwell_tools
+
+
+def start_workflow(arguments):
```

Comment:
optional: Having the same name as the `cromwell-tools.start_workflow` here brings some confusion. What do you think?

Reply:
Yes I do think it's confusing. Maybe I'll add an underscore or something to distinguish it; I'll think about whether there are more descriptive names.

Reply:
Changed to `_start_workflow` to indicate a private method and add a distinction between the calls.